source/vpn_services.json

```diff
@@ -133,11 +133,27 @@
             "MESSENGERS"
         ],
         "domains": [
+            "discord-attachments-uploads-prd.storage.googleapis.com",
```

Comment:
Hello. I'm not sure about adding these domains, they look suspicious, some don't open. 
```
discordsays.com
discordactivities.com
discordpartygames.com
discord.store
discordapp.com
discord.design
```

Reply:
discord.dev redirected to https://discord.com/developers/docs/intro
Are these domains really needed?